Makefile

```diff
@@ -288,6 +297,10 @@ inplace-postinstall-permissions:
 		chcon -R -t httpd_sys_rw_content_t $(CURDIR)/webapp/public/images; \
 		chcon    -t httpd_exec_t $(CURDIR)/lib/alert; \
 	fi
+	while [ `pwd` != "/" ]; do \
```

Comment:
Is there a reason why we need to list contents up to the root? I'd assume only starting from the installation path should be enough. I would prefer not changing permissions on system-defined directories.

Reply:
Some programs are executed using the full absolute path. If any component of that path is not traversable (`x` permission) then it will fail.
But I do agree that we should be careful modifying stuff above the installation directory, so I'd say at least a warning if changing that would be in order.

Reply:
> Is there a reason why we need to list contents up to the root? I'd assume only starting from the installation path should be enough. I would prefer not changing permissions on system-defined directories.

I needed it on my setup where `o` has no `x` bit set on any directory. We wouldn't touch system-defined directories.. we only add an additional `ACL` which allows the webserver to follow the path.

The main problem is that doing the `su` to the `www-data` to check if it has access is quite hard and might often not allowed. So we either need to make this explicit like we have here or would need to come up with a nice test which might do interactive things which is not allowed for that account.

Reply:
I'll take the `setfacl` out which goes out of the installation path. We can discuss it in another PR.